sample/src/main/assets/json.form/single_form.json

```diff
@@ -240,6 +240,30 @@
           "err": "Please enter the sex"
         }
       },
+      {
+        "key": "marital",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "person",
+        "openmrs_entity_id": "person_attribute",
+        "type": "spinner",
+        "hint": "Marital Status",
+        "values": [
+          "Ame OA",
+          "Haja OA"
+        ],
+        "alias": {
+          "Ame OA": "Married",
```

Comment:
Could we use an array here? It will be lighter and order is maintained for matching with values

Reply:
Updated to Json Array